### PR TITLE
Dataset Details update

### PR DESCRIPTION
**Pull request by wiola-dreszler**

### CODE REVIEW

File: client/src/Dataset/Details/Details.tsx [hunk 1]
```diff
@@ -56,53 +62,86 @@ const Details: React.FC<{ id: number }> = ({ id }) => {
...
+                        className={classes.drawer}
+                        variant="permanent"
+                        classes={{
+                            paper: classes.drawerPaper,
+                        }}
+                        anchor="left"
+                    >
+                        <div
+                            id="dragger"
+                            onMouseDown={(event: React.MouseEvent) => {
```
Comment (wojciech-bilicki) [inline client/src/Dataset/Details/Details.tsx]: if you pass the default argument to the handler function then there's no need to create new function
```suggestion
                            onMouseDown={handleMouseDown}
```

File: client/src/Dataset/Details/Images/Images.tsx [hunk 2]
```diff
@@ -65,7 +69,10 @@ const Images: React.FC<Props> = ({
                                     </MenuItem>
                                     <MenuItem
                                         onClick={() =>
-                                            downloadImageAction(o.id)
+                                            downloadImageAction(
```
Comment (wojciech-bilicki) [inline client/src/Dataset/Details/Images/Images.tsx]: function that takes two or more args should accept the object to use named params

File: client/src/Dataset/Details/Details.tsx [hunk 3]
```diff
@@ -56,53 +62,86 @@ const Details: React.FC<{ id: number }> = ({ id }) => {
...
+                                handleMouseDown(event);
```
Comment (maciejdrozdowski) [inline client/src/Dataset/Details/Details.tsx]: onMouseDown={handleMouseDown}

**Review by maciejdrozdowski — CHANGES_REQUESTED** [review]

File: client/src/Dataset/datasets.api.tsx [hunk 4]
```diff
@@ -89,7 +100,7 @@ export const getDetails = async ({
     return data;
 };
 
-export const generate = async (id: number, keyword: string, limit: number) => {
+export const generate = async (id: number, keyword: string, limit: string) => {
```
Comment (wojciech-bilicki) [inline client/src/Dataset/datasets.api.tsx]: function taking few args should accept object as an argument

**Review by wojciech-bilicki — CHANGES_REQUESTED** [review]

**Review by wojciech-bilicki — APPROVED** [review]

**Review by maciejdrozdowski — APPROVED** [review]

**Pull request merged**